boot/freeldr/freeldr/lib/fs/ntfs.c

```diff
@@ -54,6 +54,8 @@ typedef struct _NTFS_VOLUME_INFO
 
 PNTFS_VOLUME_INFO NtfsVolumes[MAX_FDS];
 
+BOOLEAN NoNtfsFileCaseSensitive = FALSE;
```

Comment:
Simplify a bit: Reverse values everywhere
```suggestion
BOOLEAN NtfsFileCaseSensitive = TRUE;
```

Reply:
i moved this boolean in fs.c and renamed to FSFileNameCaseSensitive

Reply:
> renamed to FSFileNameCaseSensitive

PR (code) seems to target POSIX filenames on NTFS, only, so why a "fully" generic variable name?

Also, I am somewhat surprised by the "high" number of value swaps needed.
Could you add some comments to explain these back-and-forth changes?

Reply:
I used a generic name to be able to use this boolean in other file systems. Yes, I added some comments, see if they help you understand.

Reply:
> I added some comments, see if they help you understand.

What is still unclear to me is why you want/need to reenable case sensitivity:
is FreeLdr using a mix of POSIX (sensitive) and Win32 (insensitive) APIs?

Reply:
Yes, the freeldr is using mix of POSIX and Win32 APIs.